pkg/network/infraconfigurators/bridge.go

```diff
@@ -81,17 +81,17 @@ func (b *BridgePodNetworkConfigurator) DiscoverPodNetworkInterface(podIfaceName
 	return nil
 }
 
-func (b *BridgePodNetworkConfigurator) GenerateDHCPConfig() *cache.DHCPConfig {
+func (b *BridgePodNetworkConfigurator) GenerateDHCPConfig() (*cache.DHCPConfig, error) {
```

Comment:
I don't support this change.

Bridge binding might theoretically fail - if we fail to template an hardcoded string. That cannot happen.

I think it conveys the wrong message to the reader as well - generating the config cannot fail. Retrieving the info for generating the config can.

Reply:
Changed the commit to remove the error return from `getFakeBridgeIP`.